Extensions/PlatformBehavior/PlatformerObjectRuntimeBehavior.h

```diff
@@ -43,6 +44,7 @@ class GD_EXTENSION_API PlatformerObjectRuntimeBehavior
   void SetMaxFallingSpeed(double maxFallingSpeed_) {
     maxFallingSpeed = maxFallingSpeed_;
   };
+  void SetLadderClimbingSpeed(double ladderClimbingSpeed_) { ladderClimbingSpeed=ladderClimbingSpeed_; };
```

Comment:
spacing around =

Reply:
Haha, I see why you are writing nitpicking, I made that change as well.